src/v/coproc/tests/fixtures/fiber_mock_fixture.cc

```diff
@@ -58,9 +58,9 @@ do_verify_partition(storage::log log, model::ntp ntp, std::size_t n) {
 }
 
 ss::future<> fiber_mock_fixture::verify_results(expected_t expected) {
-    return _state.invoke_on_all([this, expected](state& s) mutable {
+    return _state.invoke_on_all([this, expected](state&) mutable {
```

Comment:
Is this equivalent to `smp::invoke_on_all` since it doesn't use the sharded object?

Reply:
yeh i think they are 99% equivalent. iirc there is some subtle stuff related to priority with smp::invoke_on_all but in this case i don't think it'd make any different.